src/config_file.c

```diff
@@ -503,17 +545,24 @@ RaucSlot *find_slot_by_device(GHashTable *slots, const gchar *device)
 {
 	GHashTableIter iter;
 	RaucSlot *slot;
+	g_autofree MountableObj *obj = normalize_mountable_object(device);
+
+	if (!obj) {
```

Comment:
remove braces here. For one-line statements we normally do not use braces (according to kernel coding style)